modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc

```diff
@@ -0,0 +1,2 @@
+. Log in to Anypoint Platform.
+. In the navigation bar or the main Anypoint Platform screen, click *Access Management*.
```

Comment:
screen -> page?

Reply:
(global)

Reply:
If you make each step a partial, you have more flexibility.

Reply:
```suggestion
. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
```